jcdb-api/src/main/kotlin/org/utbot/jcdb/api/ext/KMetadataExtension.kt

```diff
@@ -52,6 +64,44 @@ val JcMethod.kmConstructor: KmConstructor?
     get() =
         enclosingClass.kMetadata?.constructors?.firstOrNull { it.signature?.name == name && it.signature?.desc == description }
 
+val JcParameter.kmParameter: KmValueParameter?
+    get() {
+        method.kmFunction?.let {
+            // Shift needed to properly handle extension functions
+            val shift = if (it.receiverParameterType != null) 1 else 0
```

Comment:
Could you please merge `main` branch and apply fix from there.

Reply:
Done